packages/devtools_app/test/integration_tests/integration.dart

```diff
@@ -352,8 +352,6 @@ class WebdevFixture {
       release ? '--release' : '--no-release'
     ];
 
-    final clean = await _runFlutter(['clean']);
```

Comment:
We talked about this over lunch:

We want to do these cleans to make sure we're building properly from nothing.

For now, if this gets the tests passing, feel free to submit, but leave a TODO and point to the issue with build_runner that you filed to go back to cleaning the build outputs.

Reply:
Embedding Jake's reply and suggestion on this one because of the latest sdk pub change for further discussion @DaveShuckerow  @jacob314:

> Ahhh this makes sense. The difference is that previously deleting the `.dart_tool` directory at any point was perfectly fine to do (pub only put precompiled binaries in there). Now that means you have to re-run pub get.
> 
> There are a few potential resolutions here:
> 
> * Make `flutter clean` delete only specific sub-directories of `.dart_tool`
>   
>   * I don't really like this option because it means there might be scenarios it doesn't help you recover from.
> * `flutter clean` could delete the `.packages` file
>   
>   * I believe this would give you a better error message from pub
> * `flutter clean` could re-run `flutter pub get`
>   
>   * This might be unexpected and cause other problems
> * Detect this scenario and output a nicer message from `pub` similar to what I suggested above.
>   
>   * Users aren't supposed to care about the `.dart_tool/` dir or anything under it so removing that from the message would be less confusing.
> 
> In general I think we should focus on the error messages from pub and improving that. I think it is totally fine and intuitive that you have to do a new `flutter pub get` after doing a `flutter clean`. The main question then is how to present that to the user.

Seems like the old behavior of pub (not requiring to do pub get before pub run) won't be readded back, so we have to find a way to remove this build directory.

Reply:
It seems like we should be running pub get.

This makes sense, particularly since we are running `pub run` right after.  If we were running `flutter run`, the tool would be running pub get for us anyway.

https://github.com/flutter/devtools/pull/1277